Source/CameraView/CameraView.swift

```diff
@@ -168,6 +164,17 @@ class CameraView: UIViewController, CLLocationManagerDelegate {
 
     if captureDevice != nil { beginSession() }
   }
+  
+  func handlePermission(granted: Bool, device: AVCaptureDevice) {
```

Comment:
@onmyway133 I generally see overuse of `self` in hyperoslo's projects. Feels like ObjC-ish

this is sufficient i think 

```
func handlePermission(granted: Bool, device: AVCaptureDevice) {
    if granted {
      self.captureDevice = device
      self.capturedDevices?.addObject(device)
    }
    
    dispatch_async(dispatch_get_main_queue()) {
      self.showNoCamera(!granted)
    }
  }```